### PR TITLE
Typo fixes for Kafka 101 tutorials

### PR DESCRIPTION
**Pull request by brianstrauch**
### Description

Fixes for typos I encountered while doing the Kafka 101 tutorials.

This PR also removes the `mkdir schema` step in the `kafka-console-consumer-producer` tutorial. The directory is actually created by the docker-compose.yml file, so that step is redundant.

### CODE REVIEW

File: _includes/tutorials/kafka-producer-application-callback/kafka/markup/dev/make-dev-file.adoc
Comment (ybyzek) [inline _includes/tutorials/kafka-producer-application-callback/kafka/markup/dev/make-dev-file.adoc:18]:
```suggestion
- `acks=all`: highest data durability guarantee, the leader broker persisted the record to its log and received acknowledgment of replication from all in-sync replicas. When using `acks=all`, it's strongly recommended to update https://kafka.apache.org/documentation/#min.insync.replicas[min.insync.replicas] as well.
```

File: _includes/tutorials/kafka-producer-application/kafka/markup/dev/make-dev-file.adoc
Comment (ybyzek) [inline _includes/tutorials/kafka-producer-application/kafka/markup/dev/make-dev-file.adoc:18]:
```suggestion
- `acks=all`: highest data durability guarantee, the leader broker persisted the record to its log and received acknowledgment of replication from all in-sync replicas. When using `acks=all`, it's strongly recommended to update https://kafka.apache.org/documentation/#min.insync.replicas[min.insync.replicas] as well.
```

File: _includes/tutorials/streams-to-table/kstreams/markup/dev/clean-up.adoc
Comment (ybyzek) [inline _includes/tutorials/streams-to-table/kstreams/markup/dev/clean-up.adoc:1]: "You're all down now!" is pretty hip, no? ;)

File: _data/tutorials.yaml
Comment (ybyzek) [inline _data/tutorials.yaml:300]:
```suggestion
  introduction: "You want to inspect/debug records written to a topic.  Each record key and value is a long and double, respectively.  In this tutorial you'll learn how to specify key and value deserializers with the console consumer."
```

File: _includes/tutorials/console-consumer-producer-avro/kafka/markup/dev/produce-topic-keys-values.adoc
Comment (ybyzek) [inline _includes/tutorials/console-consumer-producer-avro/kafka/markup/dev/produce-topic-keys-values.adoc:4]: @brianstrauch great catch

File: _includes/tutorials/kafka-consumer-application/kafka/markup/dev/make-supporting-classes.adoc
Comment (ybyzek) [inline _includes/tutorials/kafka-consumer-application/kafka/markup/dev/make-supporting-classes.adoc:39]:
```suggestion
<1> Iterate over all of the records and store each record's `value` in a `List`
```

File: _includes/tutorials/kafka-consumer-application/kafka/markup/dev/make-supporting-classes.adoc
Comment (ybyzek) [inline _includes/tutorials/kafka-consumer-application/kafka/markup/dev/make-supporting-classes.adoc:40]:
```suggestion
<2> If the `List` isn't empty, let's do something!
```

**Review by ybyzek — COMMENTED** [review]
@brianstrauch thanks for the PR, which corrects some good catches.  Left some comments/tweaks.

**Review by ybyzek — APPROVED** [review]
@brianstrauch thanks for the PR and addressing feedback.  LGTM.  I approve the PR, pending website build passes...

...Normally PRs kickoff a Semaphore run to validate the website build, not sure why that didn't happen in this case.  @awalther28 @rspurgeon -- do you happen to know?

**Comment by awalther28** [comment]
@brianstrauch @ybyzek we limit the people semaphore will kick off a build for when the PR is from a fork. Yeva we should take this offline to talk about how to move forward because it looks like our build might expose credentials.

**Comment by awalther28** [comment]
Hm let me try this /sem-approve

**Review by bbejeck — COMMENTED** [review]
Thanks for the PR Brian.  I just have one small comment, otherwise this LGTM.

I validated the changes by running the test-harness locally for the tutorials with changes to input files.

**Comment by bbejeck** [comment]
Thanks for the PR @brianstrauch !

**Comment by brianstrauch** [comment]
@bbejeck No problem!

> It's just a minor change, so I took the liberty of pushing a fix

I'm pretty sure the last two records in step 8 are actually not supposed to be there. (See https://kafka-tutorials.confluent.io/kafka-console-consumer-producer/kafka.html). They get added in step 9 and get printed later in step 10 (along with their keys).

**Pull request merged**
